packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -642,11 +502,94 @@ export class Relayer {
         } else throw receipt;
       } else throw receipt;
     } catch (error) {
-      this.logger.error({ at: "InsuredBridgeRelayer#Disputer", message: "Something errored disputing relay!", error });
+      this.logger.error({ at: "AcrossRelayer#Disputer", message: "Something errored disputing relay!", error });
     }
   }
 
-  private generateSlowRelayTx(deposit: Deposit, realizedLpFeePct: BN): TransactionType {
+  private async _processTransactionBatch(
```

Comment:
this method contains the real logic in this PR on dealing with sets of transactions. Please look at it closely!